src/shared/client/cg_api.cpp

```diff
@@ -339,6 +341,8 @@ void trap_R_AddPolysToScene( qhandle_t hShader, int numVerts, const polyVert_t *
 
 void trap_R_Add2dPolysIndexedToScene( polyVert_t* polys, int numPolys, int* indexes, int numIndexes, int trans_x, int trans_y, qhandle_t shader )
 {
+	if (!numPolys || !numIndexes) { return; }
```

Comment:
This could be just `!numIndexes`. If there are indexes but no polies, it's a mistake rather than a no-op (the indices are necessarily out of bounds).

Reply:
Done.